2.4.1/bullseye/include/entrypoint

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -e
+
+if [[ $UID == "${ASTRONOMER_UID:-1000}" ]]; then
+  # Since we need to support running tini as another user, we can't put tini in
+  # the ENTRYPOINT command, we have to run it here, if we haven't already
+  if [[ -z "$__TINIFIED" ]]; then
+    __TINIFIED=1 exec tini -- "$0" "$@"
+  fi
+else
+  __TINIFIED=1 exec gosu "${ASTRONOMER_USER}" tini -- "$0" "$@"
+fi
+
+if [[ -n "$EXECUTOR" && -z "$AIRFLOW__CORE__EXECUTOR" ]]; then
+  # Support for puckel style of defining configs
+  export AIRFLOW__CORE__EXECUTOR "${EXECUTOR}Executor"
```

Comment:
Sorry for catching this after the merge, but should this be:

```python
export AIRFLOW__CORE__EXECUTOR="${EXECUTOR}Executor"
#                             ^
```

or is it truly trying to export the two variables `AIRFLOW__CORE__EXECUTOR` (even though that is guaranteed to be blank at this point) and the variable _named_ `${EXECUTOR}Executor`?

Reply:
Ha, I guess that's been broken for a long time/forever?

Reply:
Feel free to PR a fix, though this might mean we can just remove this completely too.